src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,13 @@
-export const CommentList = () => <>Put the list here</>;
+import './CommentList.scss';
+
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <>
+    <div className="CommentList">
+      {comments.map(comment => (
+        <CommentInfo comment={comment} key={comment.id} />
+      ))}
+    </div>
+  </>
```

Comment:
the same. extra wrapper
```suggestion
    <div className="CommentList">
      {comments.map(comment => (
        <CommentInfo comment={comment} key={comment.id} />
      ))}
    </div>
```